python/gubbins/ValidateFastaAlignment.py

```diff
@@ -40,14 +43,15 @@ def does_each_sequence_have_a_name_and_genomic_data(self):
                   return False
                 if re.search('[^ACGTNacgtn-]', str(record.seq))  != None:
                   print("Error with the input FASTA file: One of the sequences contains odd characters, only ACGTNacgtn- are permitted")
+                  print(record.id)
                   return False
```

Comment:
Might be worth clarifying this error - just saying something like "record.id + ' contains a non-IUPAC character'" or similar

Reply:
Thanks Nick, I've extended that error message